src/kARAN1595/SumOfNumbersInString.java

```diff
@@ -0,0 +1,18 @@
+package kARAN1595;
+
+public class SumOfNumbersInString {
+	void sumOfNumber(String input) {
+		int total_sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				total_sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));